src/extensionFunctions.ts

```diff
@@ -599,4 +601,46 @@ export namespace extensionFunctions {
 			console.log('[' + msg + ']');
 		}
 	}
+
+	export async function validateCommandIDsInSelectedFile(didactUri: vscode.Uri) : Promise<void> {
+		if (didactUri) {
+			await vscode.commands.executeCommand(START_DIDACT_COMMAND, didactUri);
+		}
+		if (DidactWebviewPanel.currentPanel) {
+			const commands : any[] = extensionFunctions.gatherAllCommandsLinks();
+			if (commands && commands.length > 0) {
+				sendTextToOutputChannel(`Starting validation...`);
+				let allOk = true;
+				for(let command of commands) {
+					// validate all commands we found
+					const parsedUrl = url.parse(command, true);
+					const query = parsedUrl.query;
+					if (query.commandId) {
+						const commandId = utils.getValue(query.commandId);
+						if (commandId) {
+							const vsCommands : string[] = await vscode.commands.getCommands(true);
+							var filteredList : string[] = vsCommands.filter( function (command) {
+								return command === commandId;
+							});
+							if (filteredList.length === 1) {
+								// expected result - we found the command in the vscode command list
+							} else {
+								// unexpected result - let the user know
+								sendTextToOutputChannel(`--Missing Command ID ${commandId}`);
```

Comment:
if I understand well, it is based on the commands found in the installed extension. Maybe worth mentioning the two cases in the error message:
- either the commandid doesn't doesn't exist at all
- either it requires and extension which is not installed